httpd/handlers/checkIfUsernameIsAvailable.go

```diff
@@ -0,0 +1,27 @@
+package handlers
+
+import (
+	"errors"
+	"net/http"
+	"strings"
+
+	"github.com/jjcm/soci-backend/models"
+)
+
+// CheckIfURLIsAvailable - return a boolean value to see if a given URL is
+// already taken
+func CheckIfURLIsAvailable(w http.ResponseWriter, r *http.Request) {
+	requestedUsername := parseRouteParameter(r.URL.Path, "/users/username-is-available/")
```

Comment:
So I think this is right - this means that if we have `/users/username-is-available/jimbo` as the request, then requestedUsername will equal `jimbo` yea?